src/proxy/index.ts

```diff
@@ -211,7 +211,7 @@ export default class Proxy extends Router {
         const windowId    = refererDest && refererDest.windowId || void 0;
 
         if (session) {
-            session.options.referer = referer || session.options.referer;
+            if (!isIframe) session.options.referer = referer || session.options.referer;
```

Comment:
I suggest you do this as follows:
```suggestion
            if (!isIframe && referer)
                session.options.referer = referer;
```

Reply:
Sounds good, thanks